lib/penthouse/migration_context.rb

```diff
@@ -0,0 +1,112 @@
+require "active_record"
```

Comment:
The new migration context, as required by `Rails 5.2`.

This will need some work as it must act as the new module does in rails; for now it is just a placeholder